freebsd compile issue

llfuse 1.3.7, freebsd 12.1 64bit, python 3.6.10:
```
    freebsd64:   Running setup.py install for llfuse: started
    freebsd64:     Running setup.py install for llfuse: finished with status 'error'
    freebsd64:     Complete output from command /vagrant/.pyenv/versions/3.6.10/envs/borg-env/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ahdsp1rs/llfuse/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-vewk6ki7/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/.pyenv/versions/3.6.10/envs/borg-env/include/site/python3.6/llfuse:
    freebsd64:     running install
    freebsd64:     running build
    freebsd64:     running build_ext
    freebsd64:     building 'llfuse' extension
    freebsd64:     creating build
    freebsd64:     creating build/temp.freebsd-12.1-RELEASE-p3-amd64-3.6
    freebsd64:     creating build/temp.freebsd-12.1-RELEASE-p3-amd64-3.6/src
    freebsd64:     cc -pthread -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/vagrant/.pyenv/versions/3.6.10/envs/borg-env/include -I/vagrant/.pyenv/versions/3.6.10/include/python3.6m -c src/llfuse.c -o build/temp.freebsd-12.1-RELEASE-p3-amd64-3.6/src/llfuse.o -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=29 -Wall -Wextra -Wconversion -Wsign-compare -DLLFUSE_VERSION="1.3.7" -Wno-unused-function -Wno-implicit-fallthrough -Wno-unused-parameter
    freebsd64:     src/llfuse.c:55326:58: error: arithmetic on a pointer to an incomplete type 'struct pthread'
    freebsd64:         const pthread_t neg_one = (pthread_t) ((pthread_t) 0 - (pthread_t) 1), const_zero = (pthread_t) 0;
    freebsd64:                                                ~~~~~~~~~~~~~ ^
    freebsd64:     /usr/include/stdio.h:157:9: note: forward declaration of 'struct pthread'
    freebsd64:             struct pthread *_fl_owner;      /* current owner */
    freebsd64:                    ^
    freebsd64:     1 error generated.
    freebsd64:     error: command 'cc' failed with exit status 1
    freebsd64:     
    freebsd64:     ----------------------------------------
    freebsd64: Command "/vagrant/.pyenv/versions/3.6.10/envs/borg-env/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ahdsp1rs/llfuse/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-vewk6ki7/install-record.txt --single-version-externally-managed --compile --install-headers /vagrant/.pyenv/versions/3.6.10/envs/borg-env/include/site/python3.6/llfuse" failed with error code 1 in /tmp/pip-install-ahdsp1rs/llfuse/
```

--- a/src/fuse_api.pxi
+++ b/src/fuse_api.pxi
@@ -11,6 +11,9 @@ the terms of the GNU LGPL.
 
 '''
 
+from libc.stdint cimport uintptr_t
+
+
 def listdir(path):
     '''Like `os.listdir`, but releases the GIL.
 
@@ -382,6 +385,7 @@ ctypedef struct worker_data_t:
 cdef void* worker_start(void* data) with gil:
     cdef worker_data_t *wd
     cdef int res
+    cdef uintptr_t tid
     global exc_info
 
     wd = <worker_data_t*> data
@@ -393,8 +397,9 @@ cdef void* worker_start(void* data) with gil:
         session_loop(wd.buf, wd.bufsize)
     except:
         fuse_session_exit(session)
+        tid = wd.thread_id
         log.error('FUSE worker thread %d terminated with exception, '
-                  'aborting processing', wd.thread_id)
+                  'aborting processing', tid)
         res = pthread_mutex_lock(&exc_info_mutex)
         if res != 0:
             log.error('pthread_mutex_lock failed with %s',

